docs/guides/latest-updates.mdx

```diff
@@ -33,6 +33,26 @@ For more details on any of the topics covered in the blog, be sure to check [the
 
 {/* use url format /api/qiskit/release-notes/X.X to link to the specific X.X version of the release notes */}
 
+### Qiskit Runtime IBM Client 0.25.0
+
+*To see the release notes for all versions, visit the [Qiskit Runtime IBM Client release notes](/api/qiskit-ibm-runtime/release-notes).*
+
+**Highlights from the 0.25.0 (03 July 2024) version release**
+
+#### Deprecation notes
+
+- The `optimization_level` option in EstimatorV2 is deprecated. Instead, you can perform circuit optimization using the [Qiskit transpiler](transpile) or [Qiskit transpiler service](qiskit-transpiler-service).
```

Comment:
Alternatively we can say `EstimatorV2` to indicate we're talking about the class. 

```suggestion
- The `optimization_level` option in Estimator V2 is deprecated. Instead, you can perform circuit optimization using the [Qiskit transpiler](transpile) or [Qiskit transpiler service](qiskit-transpiler-service).
```